bin/pullpreview

```diff
@@ -29,12 +29,13 @@ up_opts = lambda do |o|
   o.array '--registries', 'URIs of docker registries to authenticate against, e.g. docker://username:password@ghcr.io', default: []
   o.string '--dns', 'Enable DNS support for pretty-looking URLs', default: "my.pullpreview.com"
   o.array '--ports', 'Ports to open for external access on the preview server', default: [
-    "80/tcp", "443/tcp", "1000-10000/tcp"
+    "80/tcp", "443/tcp"
```

Comment:
this is important. If you use host network in your docker containers it will mean, that all ports between 1000 and 10000 are open to the internet (including most database ports for example)

Reply:
I wonder why was this added by default in the first place. This should be a clear whitelist.